Engine/Fuego/EventQueue.h

```diff
@@ -9,6 +9,9 @@ namespace Fuego
     class FUEGO_API EventQueue
     {
     public:
+        EventQueue() = default;
+        FUEGO_NON_COPYABLE_NON_MOVABLE(EventQueue)
+
```

Comment:
Why do we need FUEGO_NON_COPYABLE_NON_MOVABLE? Does it relate to .dll export\import issue ?

Reply:
If a class will not be moved or copied it's better to delete related operators and constructors